src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends FigureDraw implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends FigureDraw {
```
інтерфейси варто імплементувати в класі Figure а у всіх решта їх можна видалити